mesos-docker/run/run.sh

```diff
@@ -47,10 +47,15 @@ CPU_HOST_PERCENTAGE=$RESOURCE_THRESHOLD
 
 SPARK_CONF_FOLDER="/etc/spark/conf"
 
-# Make sure we know the name of the docker machine. Fail fast if we don't
-if [[ ("$(uname)" == "Darwin") && (-z $DOCKER_MACHINE_NAME) ]]; then
-  echo "Undefined DOCKER_MACHINE_NAME. This variable is usually set for you when running 'docker env <machinename>'."
-  exit 1
+# Make sure we have docker installed on OSX.
+if [ "$(uname)" = "Darwin" ]
+then
+  which docker > /dev/null
```

Comment:
Suggestion to use type... http://stackoverflow.com/questions/592620/check-if-a-program-exists-from-a-bash-script